kore/src/Kore/Logger/Output.hs

```diff
@@ -270,11 +271,9 @@ swappableLogger mvar =
     worker a logAction = Colog.unLogAction logAction a
 
 defaultLogPretty :: SomeEntry -> Pretty.Doc ann
-defaultLogPretty entry =
+defaultLogPretty (SomeEntry entry) =
     Pretty.hsep
-        [ Pretty.brackets (Pretty.pretty severity)
+        [ Pretty.brackets (Pretty.viaShow . Reflection.typeOf $ entry)
```

Comment:
Do you have a sample of how this looks like, now?

Reply:
Sure.
When we have an `Entry` type for the log:
`[2019-12-06 15:46:26.950404633][DebugSolverSend] : (push 1 )`
When something was logged as a `LogMessage`:
`[2019-12-06 15:01:33.170087234][LogMessage] : [Debug] : Evaluator functionVariableAndEquals does not apply. []`

Reply:
Oops, just noticed that the `Severity` is missing from the output in one of the cases. I'll debug this.

Reply:
@vladciobanu Fixed the above in a new commit and edited the example.

Reply:
Ah, so it's literally the type name. That's reasonable. I think we need to do something about the repl: either just remove everything related to filtering by scope or replace the scope filter with a type filter.

Reply:
I agree about the type filter. I was planning to do that in a separate PR, since filtering is a different feature and filtering by scope is already unusable because we don't print any scope information to the user as of the latest master. I can move these changes to this PR if you think it would make more sense, though.